en/12_Project_Governance/08_Fixed_dependencies.md

```diff
@@ -17,10 +17,10 @@ Dependency | Fixed major release line
 [`composer/installers`](https://packagist.org/packages/composer/installers) | 2
 [`guzzlehttp/guzzle`](https://packagist.org/packages/guzzlehttp/guzzle) | 7
 [`guzzlehttp/psr7`](https://packagist.org/packages/guzzlehttp/psr7) | 2
-[`intervention/image`](https://packagist.org/packages/intervention/image) | 2
+[`intervention/image`](https://packagist.org/packages/intervention/image) | 3
 [`league/flysystem`](https://packagist.org/packages/league/flysystem) | 3
 [`monolog/monolog`](https://packagist.org/packages/monolog/monolog) | 3
-[`phpunit/phpunit`](https://packagist.org/packages/phpunit/phpunit) | 9
+[`phpunit/phpunit`](https://packagist.org/packages/phpunit/phpunit) | 11
 [`symfony/*`](https://packagist.org/packages/symfony/) | 7
 [`webonyx/graphql-php`](https://packagist.org/packages/webonyx/graphql-php) | 15
```

Comment:
Might be worth removing webonyx/graphql-php while we are here, I'm not sure why this one is listed as fixed

Reply:
I can't find the issue that spawned this page, which might explain the reasoning. I'm not fussed either way. Will leave unmerged in case you want to remove it. If you choose not to remove it, just say so and I'll merge.

Reply:
Will remove, we may well need to update it midway through the life cycle of CMS 6